doc/user_guide/user_guide.md

```diff
@@ -2,151 +2,61 @@
 
 ## Overview
 
-This project aims to create a virtual machine image, in different formats, which can be used to easily build Exasol's script-languages-container, which are the runtime container for UDF's.
+Exaol AI-Lab can be used to explore AI applications on top of Exasol database.
 
-The virtual machine image provides:
-* The script-languages-release repositories
-* All necessary dependencies to execute _exaslct_ in script-languages-release (This includes a correctly configured docker runtime)
-* A running Jupyterlab instance which is automatically started during boot of the vm
+The AI-Lab is available in multiple editions involving different technology stacks.
 
-## Objective
+Depending on your operating system, infrastructure, administrative permissions, and technical skills, you might have specific constraints and preferences.
 
-The objective is to have a complete VM ready, including all dependencies, which can be used as base to customize the existing template script-languages-container (e.g. python-3.x-minimal templates, the r-minimal templates); or, if needed, to create new script-languages-container from scratch.
-The disk of the VM is per default large enough to build and test script-languages-container.
+See [AI-Lab Editions](editions.md) to select your favorite edition.
 
-## Provided media
+<!--
+### Usage
 
-### AMI
+* [AMI Edition](ami_usage.md)
+* [VM Edition](vm_usage.md)
+* [Docker Edition](docker/docker_usage.md)
+-->
 
-The AMI id is linked in the [release notes](https://github.com/exasol/data-science-sandbox/releases/latest) and can be used to start an EC2-instance in your AWS account.
-Following name format is used for the AMI: "_Exasol-Data-Science-Sandbox-**version**_", e.g. "_Exasol-Data-Science-Sandbox-5.0.0_"
+## System Requirements
 
-### VM image
+The editions of the AI-Lab have common requirements to be available on your system:
+* CPU Architecture: x86
+* CPU cores: minimum 1, recommended 2 cores
+* Main memory (RAM): minimum 2 GiB, recommended 8 GiB
+* Network access
+* Recommend access to a running instance of Exasol database, otherwise you can start a small Exasol Docker DB
 
-Currently two VM formats are supported:
+As the AI-Lab is meant to explore AI applications on top of Exasol database, you, of course, need an instance of Exasol database running and be able to connect to it.
 
-| Format      | Description                       |
-| ----------- | --------------------------------- |
-| VMDK        | VMware Virtual Machine Disk       |
-| VHD         | Virtual Hard Disk by Microsoft    |
+AI-Lab can automatically launch such an instance on demand. However when using AI-Lab's Docker Edition there are [additional constraints](docker/docker_usage.md#enabling-exasol-ai-lab-to-use-docker-features).
```

Comment:
I have the feeling that should be part of the editions, because otherwise you are linking back and forth